robot-core/src/main/java/org/obolibrary/robot/QueryOperation.java

```diff
@@ -33,60 +39,170 @@
   private static final String queryTypeError = NS + "QUERY TYPE ERROR unknown query type: %s";
 
   /**
-   * Load an ontology into a DatasetGraph. The ontology is not changed. NOTE: This is not elegant!
-   * It basically pipes Turtle output from OWLAPI to Jena Arq.
+   * Given an ontology, return a dataset containing that ontology as the only graph.
    *
-   * @param ontology The ontology to load into the graph.
-   * @return A new DatasetGraph with the ontology loaded into the default graph.
-   * @throws OWLOntologyStorageException rarely
+   * @param ontology ontology to query
+   * @return dataset to query
+   * @throws IOException on issue creating temp files
+   * @throws OWLOntologyStorageException on issue writing ontology to TTL format
    */
-  public static DatasetGraph loadOntology(OWLOntology ontology) throws OWLOntologyStorageException {
```

Comment:
We can't drop this signature. I think the best option is to create a new method `loadOntologyDataset`.